### DISCUSSION TITLE
星空内网穿透 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#197] [模板讨论]
星空内网穿透 模板问题反馈评论区
有问题可以下面反馈。

**Comment by Goldenbeef** [comment]
执行报错：
2023-12-09 10:25:33
日志：Failed at 4/8 request,
Fail assert: {"re": "200", "from": "content"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://frp.starryfrp.com/console/

**Reply by wjf0214** [reply]
登录失败了。账号密码对吗？

**Reply by Goldenbeef** [reply]
账号密码没问题，刚刚注册的。我之前也以为输错了，从qb复制粘贴到官网是能正常登陆的。

**Reply by wjf0214** [reply]
一条一条请求看看，返回信息？
群里别人使用也是正常的，我这边复现不出来。

**Reply by Goldenbeef** [reply]
好的，那我再试试，谢谢

**Comment by skipandgoon** [comment]
Failed at 1/8 request,
Fail assert: {"re": "status=403", "from": "header-location"} from failed_asserts,
Response Error : HTTP 302: Found,
Request URL: https://frp.starryfrp.com/auth/login/login?r_u=http%3A%2F%2Ffrp.starryfrp.com%2Fconsole%3F
好像登录加验证了，求大佬修复
